dd-sdk-android/apiSurface

```diff
@@ -78,7 +78,7 @@ data class com.datadog.android.core.configuration.Configuration
     fun useCustomRumEndpoint(String): Builder
     fun trackInteractions(Array<com.datadog.android.rum.tracking.ViewAttributesProvider> = emptyArray(), com.datadog.android.rum.tracking.InteractionPredicate = NoOpInteractionPredicate()): Builder
     fun trackLongTasks(Long = DEFAULT_LONG_TASK_THRESHOLD_MS): Builder
-    fun useViewTrackingStrategy(com.datadog.android.rum.tracking.ViewTrackingStrategy): Builder
+    fun useViewTrackingStrategy(com.datadog.android.rum.tracking.ViewTrackingStrategy?): Builder
```

Comment:
should we do this or should we have it null by default in the configuration and have this method as before ?

Reply:
I think we should have it `ActivityViewTrackingStrategy` in the configuration by default, to ease onboarding.